src/AppBundle/Entity/Address.php

```diff
@@ -0,0 +1,151 @@
+<?php
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @author Vašek Boch <vasek.boch@live.com>
+ * @author Jan Klat <jenik@klatys.cz>
+ *
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\AddressRepository")
+ * @ORM\Table(name="address")
+ */
+class Address
+{
+
+	/**
+	 * @var int
+	 * @ORM\Id
+	 * @ORM\GeneratedValue
+	 * @ORM\Column(type="integer")
+	 */
+	private $id;
+
+	/**
+	 * @ORM\Column(type="string", length=255, unique=false, name="firstName")
```

Comment:
tady jdeš proti zbytku projektu, sloupec v db se bude jmenovat `firstName`, přestože jinde by to bylo `first_name`. Platí i pro ostatní sloupce